Sources/Penny/Handlers/InteractionHandler.swift

```diff
@@ -27,11 +29,29 @@ struct InteractionHandler {
     private let oops = "Oopsie Woopsie... Something went wrong :("
     
     typealias InteractionOption = Interaction.ApplicationCommand.Option
+
     
     init(event: Interaction) {
         self.event = event
         self.logger[metadataKey: "event"] = "\(event)"
     }
+
+    private func makeJWTSigners() -> JWTSigner? {
+        let signer: JWTSigner
+
+        guard let privateKeyString = Constants.accountLinkOAuthPrivKey else {
+            fatalError("Missing ACCOUNT_LINKING_OAUTH_FLOW_PRIV_KEY env var")
+        }
+
+        guard let privateKey = try? ECDSAKey.private(pem: privateKeyString) else {
```

Comment:
It should be fine for the function to throw. Change this to `try`.
The command-handling methods will automatically send an "oops" if the try fails, IIRC.